illustrations.py

```diff
@@ -844,5 +854,19 @@ def get_edge_colors(ts):
 cli.add_command(simplification)
 cli.add_command(inference)
 
+
+### JSON data below from running tskit_arg_visualizer on the tree sequences above
+
+tskit_arg_visualizer_json = dict(
```

Comment:
This is pretty nasty - maybe we could save these as individual (formatted) JSON files? Massive single line JSON dumps are not very nice for git.

Reply:
Yeah - happy to save them as (separate) plain text files. I'm not keen to format the JSON though. I just want to copy and paste from within the `tskit_arg_visualiser`. If we change them, chance are the entire JSON will change anyway, so I don't think we should worry about git diffs (and they might seem massive visually, but they are not many bytes to store)

Reply:
I think of it as git's fault if it doesn't like strings with no line breaks 😄

Reply:
OK, now as separate json files.